Dockerfile

```diff
@@ -11,15 +11,26 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 #
+FROM openzipkin/jre-full:1.8.0_72
+MAINTAINER OpenZipkin "http://zipkin.io/"
 
-FROM openzipkin/zipkin-base:base-1.40.1
+RUN apk update && apk add curl
```

Comment:
cool trick on --no-cache, should keep the image a bit smaller https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache